kernel/kernel-api/src/main/java/io/delta/kernel/TransactionCommitResult.java

```diff
@@ -44,14 +46,8 @@ public long getVersion() {
     return version;
   }
 
-  /**
-   * Is the table ready for checkpoint (i.e. there are enough commits since the last checkpoint)? If
-   * yes the connector can choose to checkpoint as the version the transaction is committed as using
-   * {@link Table#checkpoint(Engine, long)}
-   *
-   * @return Is the table ready for checkpointing?
-   */
-  public boolean isReadyForCheckpoint() {
-    return isReadyForCheckpoint;
+  /** @return list of operations to trigger after commit. */
```

Comment:
Add more details such as how the connector can choose to run and how it may affect the query performance etc.

Reply:
Added a usage section to cover these. "query performance" means the commit latency in this context?